simple_settings/special_settings.py

```diff
@@ -111,20 +111,20 @@ def configure_logging(settings_dict):
         logging.config.dictConfig(cfg)
 
 
-SPECIAL_SETTINGS_MAPPING = {
-    'OVERRIDE_BY_ENV': override_settings_by_env,
-    'REQUIRED_SETTINGS': required_settings,
-    'REQUIRED_NOT_NONE_SETTINGS': required_not_none_settings,
-    'REQUIRED_SETTINGS_TYPES': required_settings_types,
-    'CONFIGURE_LOGGING': configure_logging
-}
+SPECIAL_SETTINGS_MAPPING = (
+    ('OVERRIDE_BY_ENV', override_settings_by_env),
+    ('REQUIRED_SETTINGS', required_settings),
+    ('REQUIRED_NOT_NONE_SETTINGS', required_not_none_settings),
+    ('REQUIRED_SETTINGS_TYPES', required_settings_types),
+    ('CONFIGURE_LOGGING', configure_logging),
+)
```

Comment:
What about `collections.OrderedDict`?

Reply:
can do, but it would restrict the python version to >= 2.7 

happy to use it if you're okay with dropping support for < 2.7

Reply:
There's no problem, the min tested version of python which is compatible with simple-settings is Python 2.7